android/app/src/main/java/com/example/clarity/sdk/ClaritySDK.kt

```diff
@@ -148,6 +148,15 @@ interface API {
     @POST("toggleCardSetVisibility")
     suspend fun toggleCardSetVisibility(@Body request: ToggleCardSetRequest): Response<ToggleCardSetResponse>
 
+    @GET("getWords")
+    suspend fun getWords(): Response<GetWordsResponse>
+
+//    @POST("getProgressForSet")
```

Comment:
What was our reasoning for removing these endpoints again, are we depreciating set progress in general, or is this being transferred to a different endpoint.